src/logging.h

```diff
@@ -77,7 +77,6 @@ namespace BCLog {
         Info,      // Default
         Warning,
         Error,
-        None, // Internal use only
```

Comment:
Minor thing, but there's still a lingering reference in `contrib/devtools/bitcoin-tidy/example_logprintf.cpp`. Not sure if that needs to be updated or not.